src/structure/pure/pagejump.js

```diff
@@ -48,7 +47,15 @@ design.basicdesign.addCreator('pagejump', {
       'align'   : $e.attr('align'),
       'target'  : target
     } );
-    return ret_val + actor + '</div>';
+    var info = '';
+    var infoWidget = $('infowidget > *', $e).first()[0];
```

Comment:
Can this be solved differently as a "dependancy" of a widget of a totally different plugin usually isn't good architecture...